e2e/tests/assorted/09-savepasswordbanner.spec.js

```diff
@@ -0,0 +1,65 @@
+const {
```

Comment:
Move these tests to `01-e2eencryption.spec` with the rest of E2E Encryption tests